test/hotspot/jtreg/compiler/ciReplay/CiReplayBase.java

```diff
@@ -289,6 +307,26 @@ private String getCoreFileLocation(String crashOutputString) {
                 return coreWithoutPid;
             }
         }
+        if (Platform.isLinux()) {
+            // Maybe a systemd linux system. Try to retrieve core
+            // file. It can take a few seconds for the system to
+            // process the just produced core file so we may need to
+            // retry a few times.
```

Comment:
If this is really the case, shouldn't all the SA core file tests be running into this problem? Perhaps the following in CoreUtils.java is why they don't:
```        } else if (Platform.isLinux()) {
            // Check if a crash report tool is installed.
            File corePatternFile = new File(CORE_PATTERN_FILE_NAME);
            try (Scanner scanner = new Scanner(corePatternFile)) {
                while (scanner.hasNextLine()) {
                    String line = scanner.nextLine();
                    line = line.trim();
                    System.out.println(line);
                    if (line.startsWith("|")) {
                        System.out.println(
                            "\nThis system uses a crash report tool ($cat /proc/sys/kernel/core_pattern).\n" +
                            "Core files might not be generated. Please reset /proc/sys/kernel/core_pattern\n" +
                            "to enable core generation. Skipping this test.");
                        throw new SkippedException("This system uses a crash report tool");
                    }
                }
            }
        }
```

Reply:
Right. But I still think it's easy enough and useful to support systemd. See the updated change.